README.md

```diff
@@ -48,6 +48,9 @@ require('time-input-polyfill/auto');
 
 // ES6
 import 'time-input-polyfill/auto';
+
+// TypeScript
+import * as timePolyFill from 'time-input-polyfill';
```

Comment:
Oh sorry, this isn't actually right.

This section is taking about importing the auto application of the polyfill, not the default polyfill export.

This should just be removed. We don't need to document a Typescript import.